apps/linode-marketplace-openlitespeed-django/provision.yml

```diff
@@ -0,0 +1,23 @@
+# openlitespeed
+- name: setting up environment 
+  hosts: localhost
+  connection: local
+  any_errors_fatal: true
+  user: root
+  vars_files:
+    - group_vars/linode/vars
+  tasks:
+  # generate creds
+    - name: generating sudo user and password 
+      import_tasks: ../linode_helpers/roles/sudouser/tasks/main.yml
+      
+    - name: generating openlitespeed_password
```

Comment:
generate & writing creds in same task
 - name: writing passwords to vars
      blockinfile:
        insertafter: EOF
        path: group_vars/linode/vars
        block: | 
          openlitespeed_password: {{ lookup('password', '/dev/null length=45 chars=ascii_letters,digits') }}

Reply:
Done.